Octokit.Reactive/Clients/IObservableRepositoryContentsClient.cs

```diff
@@ -97,6 +97,30 @@ public interface IObservableRepositoryContentsClient
         IObservable<byte[]> GetArchive(string owner, string name, ArchiveFormat archiveFormat, string reference);
 
         /// <summary>
+        /// Get an archive of a given repository's contents, in a specific format
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/repos/contents/#get-archive-link</remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="archiveFormat">The format of the archive. Can be either tarball or zipball</param>
+        /// <param name="reference">A valid Git reference.</param>
+        /// <param name="timeout"> Timeout in minutes </param>
+        /// <returns>The binary contents of the archive</returns>
+        IObservable<byte[]> GetArchive(string owner, string name, ArchiveFormat archiveFormat, string reference, int timeout);
```

Comment:
Honestly, I'd rather not have this method and only have the `TimeSpan` I think it promotes better practices. Also, if we really want this overload, we can add it as an extension method to the interface.